File: Easy/JavaScript/1844. Replace All Digits with Characters.js
var replaceDigits = function(s) {
        var word = s.split("");

        for(let i = 1; i < word.length; i += 2){
            word[i] = shift(String(word[i - 1]), Number(word[i]));
        }

        return word.join("");
};

function shift(c, x) {
    return String.fromCharCode(c.charCodeAt(0) + x);
}

File: Easy/Python/1844. Replace All Digits with Characters.py
class Solution(object):
    def replaceDigits(self, s):
        word = list(s)

        for i in range(1, len(word), 2):
            word[i] = chr(ord(word[i - 1]) + ord(word[i]) - ord('0'))

        return ''.join(word)
